src/NServiceBus.Core/DetectObsoleteConfigurationSettings.cs

```diff
@@ -43,6 +44,19 @@ static void DetectObsoleteConfiguration(UnicastBusConfig unicastBusConfig)
             }
         }
 
+        static void DetectObsoleteConfiguration(AuditConfig auditConfig)
+        {
+            if (!string.IsNullOrWhiteSpace(auditConfig?.QueueName))
+            {
+                Logger.Warn($"The use of the {nameof(AuditConfig.QueueName)} attribute in the {nameof(AuditConfig)} configuration section is discouraged and will be removed in the next major version. Remove this from the configuration section. Switch to the code API by using `{nameof(EndpointConfiguration)}.AuditProcessedMessagesTo` instead.");
+            }
+
+            if (auditConfig?.OverrideTimeToBeReceived != null)
+            {
+                Logger.Warn($"The use of the {nameof(AuditConfig.OverrideTimeToBeReceived)} attribute in the {nameof(AuditConfig)} configuration section is discouraged and will be removed in the next major version. Switch to the code API by using `{nameof(EndpointConfiguration)}.AuditProcessedMessagesTo` instead.");
```

Comment:
I don't think that's necessary. There are only 2 parameters - it's not exactly a complicated method signature.